### DISCUSSION TITLE
How can I add "ease-in-out" to _hover for my transition?

### DISCUSSION THREAD

**Discussion by snezhoque** [chakra-ui/chakra-ui#6124] [Q&A] [answered]
```
<Container
      _hover={{
        shadow: 'md',
        transform: 'translateY(-5px)',
        transition: '0.2s  ̶e̶a̶s̶e̶-̶i̶n̶-̶o̶u̶t̶',
      }}
>
```

**Comment by hugobqd** [comment, marked as answer]
Hello! Split _transform_'s shorthand properties into  **transitionTimingFunction** and **transitionDuration** (Here are the defaults token, but every valid css attributes is allowed : https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/foundations/transition.ts)

```jsx
<Container
  _hover={{
    shadow: 'md',
    transform: 'translateY(-5px)',
    transitionDuration: '0.2s',
    transitionTimingFunction: "ease-in-out"
  }}
>
```

**Reply by snezhoque** [reply]
thank you!

**Reply by sleekLancelot** [reply]
Sadly this didn't work for me on backgroundGradient
```
<Button
  bgGradient={'linear(to-r, rgba(222, 13, 111, 1), rgba(115, 16, 84, 1))'}
  _hover={{
	  bgGradient: 'linear(to-l, rgba(222, 13, 111, 1), rgba(115, 16, 84, 1))',
	  transitionDuration: '2s',
         transitionTimingFunction: "ease-in-out"
  }}
  >...</Button>
```

**Reply by rishinopany** [reply]
Same, it isn't working for me either for bgGradient.

**Reply by bstrihova** [reply]
because bgGradient is actually a css property of background-image, background-image is not animatable, please see https://stackoverflow.com/questions/19808764/transition-for-background-image-in-firefox for other possible solutions

**Reply by sneh03-a** [reply]
How can I add transition here ?

_hover={{
        backgroundSize:"110%",
      }}

**Reply by bstrihova** [reply]
> How can I add transition here ?
> 
> _hover={{ backgroundSize:"110%", }}

please provide more context and code, it's hard to understand your question
